packages/zoe/tools/manualTimer.js

```diff
@@ -72,38 +106,22 @@ export default function buildManualTimer(log, startValue = 0) {
       return harden(baseTimes);
     },
     createRepeater(delay, interval) {
```

Comment:
should this be `delaySecs` to match the old, actual implementation?

Reply:
No, it should be `delay` to match the old, deprecated typescript declaration.